src/client/pfs/pfs.go

```diff
@@ -1,8 +1,36 @@
 package pfs
 
-import "fmt"
+import (
+	"crypto/sha512"
+	"encoding/base64"
+	"encoding/hex"
+	"fmt"
+	"hash"
+)
+
+var (
+	// ChunkSize is the size of file chunks when resumable upload is used
+	ChunkSize = int64(16 * 1024 * 1024) // 16 MB
```

Comment:
Shouldn't this be configurable?
In fact, if `ChunkSize` is a constant it can waste some bandwidth if the remaining chunk is the last chunk and it is less than 16MB.
It'd be useful to have the chunk size as part of the protocol.

Reply:
I'm not sure if I understand.  It's true that the last chunk may be less than 16MB, but how does it waste bandwidth?

Reply:
That comment made me think that the code requires all chunks to be 16MB so if the last chunk is retransmitted it has to be exactly 16MB so some data is retransmitted and overwritten although it is already there.